packages/table/src/quadrants/tableQuadrantStack.tsx

```diff
@@ -71,7 +72,9 @@ export interface ITableQuadrantStackProps extends IProps {
     isHorizontalScrollDisabled?: boolean;
 
     /**
-     * If `false`, hides the row headers and settings menu.
+     * If `false`, hides the row headers and settings menu. Affects the layout
+     * of the table, so we need to know when this changes change in order to
```

Comment:
extra word? "when this changes in order to"

Reply:
👍